portal/hooks/useEstimateFees.ts

```diff
@@ -24,8 +24,7 @@ type OperationsToEstimate =
   | 'unstake'
   | 'withdraw-btc'
 
-const GasUnitsEstimations: Record<OperationsToEstimate, bigint> = {
-  'approve-erc20': BigInt(45_000),
+const GasUnitsEstimations: Partial<Record<OperationsToEstimate, bigint>> = {
```

Comment:
I think we can then remove `approve-erc20` from `OperationsToEstimate` in line 20, right?